.github/pull-request-template.md

```diff
@@ -17,3 +17,8 @@ Changes proposed in this pull request:
 
 **Related issue(s)**
 <!-- If you refer to a particular issue, provide its number. For example, `Resolves #123`, `Fixes #43`, or `See also #33`. -->
+
+**Confirm that everything was done**
+- [ ] Related issues are linked
```

Comment:
What about issues created in our private repo, should I link them as well?

Reply:
@koala7659  I've made a suggestion about that point in 10ffc1d40e524bc5cc114563f81604dfbfc25a63